plugins/module_utils/rds.py

```diff
@@ -185,8 +185,17 @@ def handle_errors(module, exception, method_name, parameters):
         accepted_engines = [
             'aurora', 'aurora-mysql', 'aurora-postgresql'
         ]
+        accepted_engine_modes = [
```

Comment:
Isn't this redundant since we already validate the user inputs with `argument_spec`.

Reply:
I can't remember the exact reason behind it now. I guess we can remove these now https://github.com/ansible-collections/amazon.aws/pull/941/files#diff-93f87b21085d00540e096e681c0d162bcc8575fe1c5995fc4b11623df4c6622dR191-R194 as long as we validate the specification.

Reply:
With `engine`, `engine_mode` and `engine_version` There is a large range of potential combinations. The API returns a clear message and I would prefer to just rely on this. Also, in the future AWS will probably adjust some of these constraints, if we've got a strict check system we will have to release a new collection version.